src/components/Loading.react.js

```diff
@@ -53,7 +55,13 @@ export default class Loading extends Component {
         const Spinner = isLoading && getSpinner(spinnerType);
 
         return (
-            <div style={isLoading ? hiddenContainer : {}}>
+            <div
+                className={isLoading ? null : className}
+                style={mergeRight(
+                    isLoading ? hiddenContainer : {},
+                    parent_style
+                )}
```

Comment:
```suggestion
                style={isLoading
                    ? mergeRight(hiddenContainer, parent_style)
                    : parent_style
                }
```
Minor, just to avoid having to do a noop merge.